Index: api/fedora_collection.inc
===================================================================
--- api/fedora_collection.inc	(revision 894)
+++ api/fedora_collection.inc	(working copy)
@@ -79,7 +79,7 @@
                              where ($object <dc:title> $title
                              and $object <fedora-model:hasModel> $content
                              and (';
-  $query_string .= $relationship;
+  // $query_string .= $relationship;
   if (is_array($relationship)) {
     foreach ($relationship as $rel) {
       $query_string .= '$object <fedora-rels-ext:'. $rel . '> <info:fedora/'. $collection_pid . '>';
